sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2525,14 +2525,19 @@ class Dataset[T] private[sql](
 
   /**
    * Returns a new Dataset that contains only the unique rows from this Dataset.
-   * This is an alias for `distinct`.
+   * This is an alias for `distinct` on batch [[Dataset]]. For streaming [[Dataset]], it would show
```

Comment:
It is an alias, `Dataset.distinct` calls `dropDuplicates`.

If we want to highlight the different between SQL distinct and `Dataset.distinct`, we should update document of `Dataset.distinct`.